schemas/+aod/trackMotion3D.m

```diff
@@ -0,0 +1,86 @@
+
+function [x,y,z,t] = trackMotion3D(fn, analysischan) 
```

Comment:
We have to run motion detection during realtime acq as well as during offline analysis. Because realtime implementation came first, the code is residing in folder structure that we have for realtime stuff. We can rearrange if it is absolutely necessary, for sure we dont want to duplicate the same code in different folders